aspnet/fundamentals/configuration/sample/src/ConfigConsole/ConfigConsole.xproj

```diff
@@ -4,17 +4,15 @@
     <VisualStudioVersion Condition="'$(VisualStudioVersion)' == ''">14.0</VisualStudioVersion>
     <VSToolsPath Condition="'$(VSToolsPath)' == ''">$(MSBuildExtensionsPath32)\Microsoft\VisualStudio\v$(VisualStudioVersion)</VSToolsPath>
   </PropertyGroup>
-
   <Import Project="$(VSToolsPath)\DNX\Microsoft.DNX.Props" Condition="'$(VSToolsPath)' != ''" />
   <PropertyGroup Label="Globals">
     <ProjectGuid>8ee63173-bff1-4e78-ab94-49a5be0eb2d8</ProjectGuid>
     <RootNamespace>ConfigConsole</RootNamespace>
     <BaseIntermediateOutputPath Condition="'$(BaseIntermediateOutputPath)'=='' ">..\..\artifacts\obj\$(MSBuildProjectName)</BaseIntermediateOutputPath>
-    <OutputPath Condition="'$(OutputPath)'=='' ">..\..\artifacts\bin\$(MSBuildProjectName)\</OutputPath>
+    <OutputPath Condition="'$(OutputPath)'=='' ">.\bin\</OutputPath>
```

Comment:
> Does this matter?

Not sure.

> What would I update it to, .\obj\ ?

In my PR, I reverted the change, but it's possible VS will want to change it again.
So I guess changing `BaseIntermediateOutputPath` to `.\obj` (which seems to be the default for new projects) makes sense.